swagger.yaml

```diff
@@ -52,6 +52,23 @@ parameters:
     schema:
       $ref: "#/definitions/TopicUpdate"
 paths:
+  /topics/:
+    get:
+      tags:
+        - "Public"
+      summary: "Get a lists of topics"
+      description: "gets a public list of top level root topics"
+      produces:
+        - "application/json"
+      responses:
+        200:
+          description: "Json object containing a list of all available topics"
+          schema:
+            $ref: '#/definitions/ListOfTopics'
+        404:
+          description: "Topic not found"
```

Comment:
I don't think this is accurate but I'm not sure what to put instead. What causes it to give a 404? I'm not sure it can!